sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -4589,13 +4589,15 @@ pi_result cuda_piextUSMEnqueueMemcpy(pi_queue queue, pi_bool blocking,
   return result;
 }
 
+// flags is currently ignored as it is a placeholder for future features
 pi_result cuda_piextUSMEnqueuePrefetch(pi_queue queue, const void *ptr,
                                        size_t size,
                                        pi_usm_migration_flags flags,
                                        pi_uint32 num_events_in_waitlist,
                                        const pi_event *events_waitlist,
                                        pi_event *event) {
-  assert(queue != nullptr);
+  assert(!(flags & ~PI_USM_MIGRATION_TBD0), PI_INVALID_VALUE)
```

Comment:
This will not work. You're missing a `;` and `assert` does not take two parameters. I think what you're after is

```c++
if (flags & ~PI_USM_MIGRATION_TBD0)
  return PI_INVALID_VALUE;
```
which is roughly what `PI_ASSERT` in pi_level_zero.cpp evaluates to.

Reply:
Sorry, silly mistake, should be fixed now